tests/dags/test_dag_serialization.py

```diff
@@ -0,0 +1,308 @@
+    'example_gcp_function',
+    'example_gcp_natural_language',
+    'example_gcp_spanner',
+    'example_gcp_speech',
+    'example_gcp_sql',
+    'example_gcp_sql_query',
+    'example_gcp_transfer',
+    'example_gcp_translate',
+    'example_gcp_video_intelligence',
+    'example_gcp_vision_autogenerated_id',
+    'example_gcp_vision_explicit_id',
+    'example_gcp_vision_annotate_image',
+    'example_gcs_acl',
+    'example_gcs_to_bq_operator',
+    'example_kubernetes_executor',
+    'example_kubernetes_executor_config',
+    'example_kubernetes_operator',
+    'pubsub-end-to-end',
+    'example_qubole_operator',
+    'example_qubole_sensor',
+    'example_twitter_dag',
+    'POC_winrm_parallel'
+]
+
+# TODO(coufon): to remove useless fields.
+serialized_simple_dag_ground_truth = (
+    '{"__type": "dag", '
+    '"__var": {'
+    '"default_args": {"__var": {}, "__type": "dict"}, '
+    '"params": {"__var": {}, "__type": "dict"}, '
+    '"_dag_id": "simple_dag", '
+    '"_full_filepath": "", '
+    '"_concurrency": 16, '
+    '"_description": "", '
+    '"fileloc": null, '
+    '"task_dict": {"__var": '
+    '{"simple_task": {"__var": {"'
+    'task_id": "simple_task", '
+    '"owner": "airflow", '
+    '"email_on_retry": true, '
+    '"email_on_failure": true, '
+    '"start_date": {"__var": "2019-08-01T00:00:00+00:00", "__type": "datetime"}, '
+    '"trigger_rule": "all_success", '
+    '"depends_on_past": false, '
+    '"wait_for_downstream": false, '
+    '"retries": 0, '
+    '"queue": "default", '
+    '"pool": "default_pool", '
+    '"retry_delay": {"__var": 300.0, "__type": "timedelta"}, '
+    '"retry_exponential_backoff": false, '
+    '"params": {"__var": {}, "__type": "dict"}, '
+    '"priority_weight": 1, '
+    '"weight_rule": "downstream", '
+    '"executor_config": {"__var": {}, "__type": "dict"}, '
+    '"do_xcom_push": true, '
+    '"_upstream_task_ids": {"__var": [], "__type": "set"}, '
+    '"_downstream_task_ids": {"__var": [], "__type": "set"}, '
+    '"inlets": [], '
+    '"outlets": [], '
+    '"_inlets": {"__var": {"auto": false, "task_ids": [], "datasets": []}, "__type": "dict"}, '
+    '"_outlets": {"__var": {"datasets": []}, "__type": "dict"}, '
+    '"_dag": {"__type": "dag", "__var": "simple_dag"}, '
+    '"ui_color": "#fff", '
+    '"ui_fgcolor": "#000", '
+    '"template_fields": [], '
+    '"_task_type": "BaseOperator"}, '
+    '"__type": "operator"}}, '
+    '"__type": "dict"}, '
+    '"timezone": {"__var": "UTC", "__type": "timezone"}, '
+    '"schedule_interval": {"__var": 86400.0, "__type": "timedelta"}, '
+    '"_schedule_interval": {"__var": 86400.0, "__type": "timedelta"}, '
+    '"last_loaded": null, '
+    '"safe_dag_id": "simple_dag", '
+    '"max_active_runs": 16, '
+    '"orientation": "LR", '
+    '"catchup": true, '
+    '"is_subdag": false, '
+    '"partial": false, '
+    '"_old_context_manager_dags": []}}')
+
+
+def make_example_dags(module, dag_ids):
```

Comment:
Please add type annotations for public functions

Reply:
Is it also required for functions in unit tests?